ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -56,21 +89,23 @@ export class AppVersionsTableContainer extends React.Component {
 
   getTable = (versions = []) => {
     const renderedRows = [];
-    for (let i = 0; i < versions.length; i++) {
+    versions.map(version => {
```

Comment:
```suggestion
    versions.forEach(version => {
```
(`forEach` makes more sense seeing as we're not making use of the returned array that `.map` creates)